src/components/dialogs/commons/notBuiltNodeWarning.js

```diff
@@ -0,0 +1,23 @@
+import Grid from '@mui/material/Grid';
+import Alert from '@mui/material/Alert';
+import { FormattedMessage } from 'react-intl';
+import React from 'react';
+import { BUILD_STATUS } from '../../network/constants';
+
+const NotBuiltNodeWarning = ({ buildStatus }) => {
+    const isNodeNotBuilt = buildStatus === BUILD_STATUS.NOT_BUILT;
+
+    return (
+        <Grid container spacing={2} paddingTop={2}>
+            <Grid item container xs={12}>
+                {isNodeNotBuilt && (
+                    <Alert severity={'warning'}>
+                        <FormattedMessage id="ModifyNodeNotBuiltWarningMsg" />
+                    </Alert>
+                )}
+            </Grid>
```

Comment:
When there is no warning, we still include an empty grid container, which increases the form's height